content/intro-to-storybook/react-native/en/test.md

```diff
@@ -0,0 +1,129 @@
+---
```

Comment:
Chromatic doesn't support React Native out of the box. Perhaps if we can get a web build going we could test that. Would that make sense for a RN user do you think @jonniebigodes?

PS I've resubscribed your personal Chromatic account on our open source plan ;)

Reply:
I'm working on the web build, I think it would be a good addition to get the full experience the other audiences have, have almost if not all the capabilities offered. Thank you very much for the subscription.